facility-manager.html

```diff
@@ -0,0 +1,806 @@
+
+<p>Examples of system specifications and set-ups for production OMERO servers have kindly been provided by a number of institutes to serve as illustrations of working systems. These are shown in the <a href="http://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/server-setup-examples.html" target="_blank" class="external-url">Example production server set-ups</a> page on the OME website.</p>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="using"></a>Using OMERO</h2>
+
+<p>Details on downloading the OMERO clients and accessing the OMERO server are in <a href="http://help.openmicroscopy.org/getting-started-5.html" target="_blank" class="external-url">Getting Started with OMERO</a> in User Help.</p>
+
+<p>Details on accessing OMERO with the OMERO.web client are in the <a href="http://help.openmicroscopy.org/web-client.html" target="_blank" class="external-url">Using the OMERO.web Client</a> section of User Help.</p>
+
+<p>Details of the command line interface <a href="http://www.openmicroscopy.org/site/support/omero5.2/users/cli/index.html" target="_blank" class="external-url">OMERO.cli</a> are in the OME documentation.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="permissions"></a>Permissions on OMERO</h2>
+
+<p>There are three levels of user permission in OMERO, Administrator, Group Owner and Member. Each level of permission allows different actions to be performed on a user's own, or other user's data, and certain administrative tasks.</p>
+
+<p>In brief, and practically speaking, Administrators can do anything they like to any user's data, in any group (except annotate in Private groups) and Group Owners can do anything to any user's data in the groups they own.</p>
+
+<p>Full details of the permissions for each level of user are shown in the <a href="http://help.openmicroscopy.org/sharing-data.html#roles" target="_blank" class="external-url">User Roles</a> table in the Sharing data section of User Help.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="access"></a>Access control, authentication, user accounts</h2>
+
+<p>The recommended way to control access to the OMERO server is to configure integration with your institute's authentication system (most commonly LDAP).</p>
+
+<p>See the documentation for <a href="http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/server-ldap.html" target="_blank" class="external-url">LDAP configuration</a> for details. Instructions for using Active Directory are at the bottom of this page.</p>
```

Comment:
"…at the bottom of **this** page" implies this help guide, "…are also included" might be less confusing.